.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -58,6 +58,16 @@ For UI changes, Please provide the Documetation PR on [info_grid](https://github
 - Can this code become too complex to understand for other devs?
 - Can this code cause future integration problems?
 
+### Testing Checklist
+
+- [ ] Meet the specified acceptance criteria.
+- [ ] Test if can affect any other functionality.
+- [ ] Tested with unit, integration, and end-to-end tests.
+- [ ] Tested the un-happy path and Rollback scenarios.
+- [ ] Documentation updated to meat the latest changes.
```

Comment:
meet*

Reply:
copy paste lelasaf :D

will fix the line copied from too :D

https://github.com/threefoldtech/tfgrid-sdk-ts/blob/2f2d755f8845d0af2897450dc4b9d5f3d4c35eb5/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L27